spec/std/process_spec.cr

```diff
@@ -85,37 +85,32 @@ describe Process do
     $?.exit_code.should eq(0)
   end
 
-  it ".effective_user_id" do
-    Process.effective_user_id.should eq `id -u`.to_i
-    Process.effective_user_id.should eq Process.effective_user_id
+  it ".user_id" do
+    Process.user_id.should eq `id -u`.to_i
+    Process.user_id.should eq Process.effective_user_id
   end
 
   it ".group_id" do
     Process.group_id.should eq `id -g`.to_i
     Process.group_id.should eq Process.effective_group_id
   end
 
-  it ".become_user" do
-    success = false
-    ruid = Process.user_id
-    euid = Process.effective_user_id
+  if euid == 0
+    it ".become_user" do
```

Comment:
The essential part was adding the scopes `as root` and `as non-root` to the example descriptions. Otherwise there is really no difference to having a single example.